app/views/responsible_body/mobile/extra_data_requests/new.html.erb

```diff
@@ -0,0 +1,18 @@
+<%- content_for :before_content do %>
```

Comment:
This page needs a `<title>` set, probably using the `title_with_error_prefix` method.

Reply:
Now done